docs/src/domains/steps/non-production-custom-domains.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Set up a custom domain on your non-production environments
+sidebarTitle: (Beta) Non-production custom domains
+weight: 2
+description: Learn how to set up custom domains on your staging and development environments
+betaFlag: true
+---
+
+You can add a custom domain to a staging or development environment without modifying your [routes configuration](../../define-routes/_index.md).
+To do so, you need to attach the new non-production custom domain
```

Comment:
Thank you for your input @chadwcarlson.
This is definitely one of the things where we are struggling to find the right wording...
I can see what you are trying to do, but I feel like the second phrase is still a bit confusing.

Maybe something like this:
> A custom non-production domain configured on your project replaces the production domain for the corresponding environment, instead of the domain automatically generated by Platform.sh. If you have multiple production domains, you will have to select which one you are replacing when adding  a custom non-production domain.

Still wordy... and we are back with using "replace" instead of "attach".
So I don't know if this is helping 🙈

Reply:
Definitely helps, thanks